contributing.md

```diff
@@ -120,7 +120,8 @@ Just as in the **subject**, use the imperative, present tense: "change" not "cha
 ## Dependency Management
 Dependencies are automatically updated by [Dependabot](https://dependabot.com/) which is now integrated
 directly into GitHub.  Each week Dependabot will create a large number of individual PRs that update
-each dependency.  To merge those into main, use the following process:
+each dependency in each of the [Experimenter](https://github.com/mozilla/experimenter) and
+[Experimenter Docs](https://github.com/mozilla/experimenter) repos.  To merge those into the `main` branch, use the following process:
```

Comment:
No wait, this still needs to update to "https://github.com/mozilla/experimenter-docs"

Reply:
Omg how did you re request change so fast when I have auto merge enabled XD  Pdehaan: fastest unapprove in the west XD